source/api/net/sslencoding.rst

```diff
@@ -0,0 +1,19 @@
+net.SslEncoding namespace
+#########################
+
+Contains constants representing encodings of private keys and certificates.
```

Comment:
Perhaps worth a simple example showing that they're referenced like this: `net.SslEncoding.PEM`.